src/app/dashboard-view/dashboard-view.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="dashboard-view">
+  <div class="container" *ngFor="let user of users">
+    <div class="container-content">
+      <h4 class="user-name">{{ user.firstName + " " + user.lastName }}</h4>
```

Comment:
Please use a `pipe` for displaying the full name of the user.

Reply:
I have made use of the `shorten` pipe to display the username at the end of the project [in pr#5]. Should I add it here too?

Reply:
1. Please link and fix the formatting for `[in pr#5]`.
2. For this case you can skip it but the general idea you need to follow when implementing the components is that you should do everything you are implementing to be implemented the correct way. In this case since you are displaying the user names, you should have implemented the pipe too.
3. Depending on how the pipe is actually implemented `shorten` can be the wrong name for it. Something like `username` would be more appropriate.

Reply:
> 1. Please link and fix the formatting for `[in pr#5]`.
I didn't quite understand what this means.

> 2. For this case you can skip it but the general idea you need to follow when implementing the components is that you should do everything you are implementing to be implemented the correct way. In this case since you are displaying the user names, you should have implemented the pipe too.
I will keep that in mind going forward.

> 3. Depending on how the pipe is actually implemented `shorten` can be the wrong name for it. Something like `username` would be more appropriate.
I will change the name of the pipe accordingly.

Reply:
Whenever you are providing reference to content a different url please provide the link to the same. This helps the other person to easily navigate to that link. In this case the correct syntax is:-  

> at the end of the project in [PR#5](https://github.com/nihal-rg/Angular-exercise/pull/5)

Reply:
The change has been made.